pkg/storaged/dialog.jsx

```diff
@@ -1178,7 +1186,24 @@
         </div>);
 };
 
-export function init_active_usage_processes(client, usage) {
+export function teardown_danger_message(usage, expect_single_unmount) {
+    if (is_expected_unmount(usage, expect_single_unmount))
+        return stop_processes_danger_message(usage[0].users);
+
+    const usage_with_users = usage.filter(u => u.users);
+    const n_processes = usage_with_users.reduce((sum, u) => sum + u.users.filter(u => u.pid).length, 0);
+    const n_services = usage_with_users.reduce((sum, u) => sum + u.users.filter(u => u.unit).length, 0);
+    if (n_processes > 0 && n_services > 0)
+        return _("Related processes and services will be forcefully stopped.");
+    else if (n_processes > 0)
+        return _("Related processes will be forcefully stopped.");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19388-20230929-123934-64a9fe0d-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)